### PR TITLE
Add dist/feathers.min.js

### LINKED ISSUE #64: feathers.min.js?
It'd be good to have a feathers.min.js in dist, for those of us using npmcdn. 😸

### PR DESCRIPTION
**Pull request by daffl**
Closes #64

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,8 +27,9 @@
     "clean": "rm -rf dist/ && mkdir -p dist",
     "browserify:test": "browserify test/browser/test.js -t babelify --outfile dist/browser.test.js",
     "browserify:dist": "browserify src/client.js -t babelify --standalone feathers --outfile dist/feathers.js",
-    "browserify": "npm run browserify:dist && npm run browserify:test",
-    "add-dist": "npm run clean && npm run browserify:dist && git add dist/ --force && git commit -am \"Updating dist\"",
+    "browserify:min": "browserify src/client.js -t babelify --standalone dist/ | uglifyjs > dist/feathers.min.js",
```
Comment (Qard) [inline package.json]: Looks like that `dist/` after the `--standalone` needs to be `feathers`. 1.4.0 is broken now. 😞

Comment (daffl) [inline package.json]: I just tried 1.4 and it seems to be working fine although the `--standalone` flag probably really isn't needed.

Comment (Qard) [inline package.json]: The standalone flag makes a global of the given name. In the old version, it created window.feathers. 1.4.0 creates window.dist. I'm using rollupjs to bundle my app code, which translates external ES6 module imports into references to globals, which just broke.

Comment (daffl) [inline package.json]: That makes sense. The export is fixed in v1.4.1

Comment (Qard) [inline package.json]: Great, thanks! 🎉

**Pull request merged**